routes/api.php

```diff
@@ -14,6 +16,12 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+// Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
+//     return $request->user();
+// });
+
+Route::prefix('airline')->group(function () {
+    Route::post('/', CreateAirlineController::class);
+    Route::delete('/{airlineId}', DeleteAirlineController::class);
+    Route::put('/{airline}', UpdateAirlineController::class);
```

Comment:
```suggestion
Route::prefix('airlines')->group(function () {
    Route::post('/', CreateAirlineController::class);
    
    Route::prefix('{airline}')->group(function () {
        Route::put('/', UpdateAirlineController::class);
        Route::delete('/', DeleteAirlineController::class);
    });
```
Instead of pushing commented code, it's better to delete it altogether.

You can group by prefix, too, even when using Route Model Binding.

Note that I changed the prefix of the parent group of all of these routes from `airline` to `airlines`. I did this because the prefixes tend to be plural instead of singular.